cmd/fyne/internal/commands/build.go

```diff
@@ -357,3 +371,14 @@ func appendEnv(env *[]string, varName, value string) {
 
 	*env = append(*env, varName+"="+value)
 }
+
+func getEnv(env []string, varName string) string {
+	for _, e := range env {
+		keyValue := strings.SplitN(e, "=", 2)
+
+		if keyValue[0] == "GOFLAGS" {
```

Comment:
Surely this should be `keyValue[0] == varName` ?

Reply:
Oh, good catch.